src/components/NewCommentForm.tsx

```diff
@@ -59,13 +59,10 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
     setSubmitting(true);
 
-    // it is very easy to forget about `await` keyword
     await onSubmit({ name, email, body });
 
-    // and the spinner will disappear immediately
     setSubmitting(false);
```

Comment:
Move 'setSubmitting(false);' inside a finally block to ensure it's always called.